src/server/network.ts

```diff
@@ -263,6 +264,24 @@ export type ResourceTiming = {
   responseStart: number;
 };
 
+export type IPAddressAndPort = {
```

Comment:
I think we can call it RemoteAddr, at least in the Go world, this commonly get's used for ip+port.

Reply:
👍 I'll address in next changeset incorporating any comments from Joel.